SimTKcommon/tests/StateTest.cpp

```diff
@@ -250,7 +190,6 @@ int main() {
 
 
     SimTK_START_TEST("StateTest");
-        //SimTK_SUBTEST(testLowestModified);
```

Comment:
I didn't remove them. They were already commented out because they were tests for long-ago dead code.